administrator/modules/mod_login/tmpl/default.php

```diff
@@ -13,12 +13,20 @@
 use Joomla\CMS\Language\Text;
 use Joomla\CMS\Router\Route;
 
-$app->getDocument()->getWebAssetManager()
-	->useScript('core')
+/** @var Joomla\CMS\WebAsset\WebAssetManager $wa */
+$wa = $app->getDocument()->getWebAssetManager();
+
+// Allow to template to provide own asset for the module
```

Comment:
```suggestion
// Allow template to provide its own asset for the module
```

and the same for all the other instances